github/teams_members.go

```diff
@@ -20,12 +20,12 @@ type TeamListTeamMembersOptions struct {
 	ListOptions
 }
 
-// ListTeamMembers lists all of the users who are members of the specified
-// team.
+// ListTeamMembersByID lists all of the users who are members of a team, given a specified
+// organization ID, by ID.
```

Comment:
For all of these endpoints where it says "organization ID, by ID" could you please add in the word "team"
so that it reads "organization ID, by team ID"?  I think this would be clearer.

Likewise, where it says "organization name, by slug", could you say "organization name, by team slug"?

Reply:
sure i'll do the modification